packages/js/components/src/experimental-select-control/menu.tsx

```diff
@@ -16,9 +17,24 @@ type MenuProps = {
 };
 
 export const Menu = ( { children, getMenuProps, isOpen }: MenuProps ) => {
+	const [ boundingRect, setBoundingRect ] = useState< DOMRect >();
+	const selectControlMenuRef = useRef< HTMLDivElement >( null );
+
+	useEffect( () => {
+		if (
+			selectControlMenuRef.current &&
+			selectControlMenuRef.current?.parentElement
+		) {
+			setBoundingRect(
+				selectControlMenuRef.current?.parentElement.getBoundingClientRect()
+			);
+		}
+	}, [ selectControlMenuRef.current ] );
+
+	/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events */
 	return (
-		<ul
-			{ ...getMenuProps() }
+		<div
+			{ ...getMenuProps( { ref: selectControlMenuRef } ) }
```

Comment:
This breaks accessibility and moves the `role="listbox"` to the div instead of the ul.  Items will not be read aloud in screen readers.

Is there a reason we need to move the menu props to the div?

Reply:
I was afraid of that, when it was on the other element `downshift` showed an error that the `ref` was not used correctly. My guess is because of it being inside of the Popover and thus rendered at a later time, I can see if I can change that, or leave the `getMenuProps` on the nested element but pass in the ref of the parent element (as I am not sure what the `ref` will exactly be used for.

Reply:
@joshuatf I guess the main issue was that I was conditionally rendering the `popover` previously. Apparently as part of accessibility ( [see comment](https://github.com/downshift-js/downshift/issues/1272#issuecomment-1236290430)) the menu should always be rendered.
I moved the `getMenuProps` back down and removed the `isOpen` condition and moved that down to the children (as that is fine to be conditionally rendered).
This should work as expected again.

Reply:
This is the commit btw: https://github.com/woocommerce/woocommerce/pull/34967/commits/a12b0c066cab1cff9063b8e2185ba7439a668020